### ISSUE TITLE
Could you release the code for data label generation?

### ISSUE THREAD

**Issue by Devoe-97** [xuelunshen/gim#8]

**Comment by xuelunshen** [comment]
Yes, we will definitely release the code for "data label generation." We are currently working on it. Since the `gim` contain a amount of code, we need some time to prepare it. We hope you can be patient and wait for a little while. Thank you for your interest and attention.

**Comment by Vincentqyw** [comment]
> Yes, we will definitely release the code for "data label generation." We are currently working on it. Since the `gim` contain a amount of code, we need some time to prepare it. We hope you can be patient and wait for a little while. Thank you for your interest and attention.

Do you have any updates on when the data label generation code will be released?

**Comment by xuelunshen** [comment]
@Vincentqyw Definitely to be released next month (December 2024). Thank you for your attention and patience.

**Comment by xuelunshen** [comment]
@Devoe-97 @Vincentqyw 
Sorry for the long wait, I have now released the code for video data generation. Please refer to the [README](https://github.com/xuelunshen/gim#%EF%B8%8F-video-preprocess) for relevant information.

**Comment by Vincentqyw** [comment]
@xuelunshen  That's fantastic! Thanks for your hard work!

**Comment by xuelunshen** [comment]
@Vincentqyw Thank you for recognizing our work. If you have any questions, feel free to open an issue. 😁

**Comment by Vincentqyw** [comment]
```
FileNotFoundError: [Errno 2] No such file or directory: 'weights/decoder_epoch_20.pth'
```
Hi, could you please provide the `decoder_epoch_20.pth` file from the weights folder?

**Comment by xuelunshen** [comment]
@Vincentqyw 
OK, I have updated the [README](https://github.com/xuelunshen/gim#%EF%B8%8F-video-preprocess).
